### PR TITLE
feat(flamingo): add optional attribute to configure a session key prefix for REDIS (flamingo.session.redis.keyPrefix)

### PR DESCRIPTION
**Pull request by mkrill**
…

### CODE REVIEW

File: framework/flamingo/sessions.go [hunk 1]
```diff
@@ -55,6 +57,7 @@ func (m *SessionModule) Inject(config *struct {
 	Path                 string  `inject:"config:flamingo.session.cookie.path"`
 	RedisURL             string  `inject:"config:flamingo.session.redis.url"`
 	RedisHost            string  `inject:"config:flamingo.session.redis.host"`
+	RedisKeyPrefix       string  `inject:"config:flamingo.session.redis.keyPrefix,optional"`
```
Comment (karstenkoehler) [inline framework/flamingo/sessions.go:60]: Could you add the key to the `CueConfig` method for validation and also document it in the `SessionReadme.md` file?

**Review by karstenkoehler — CHANGES_REQUESTED** [review]
Hi @mkrill, thanks for your contribution. Could you please have a look at the changes, which I proposed?

File: framework/flamingo/sessions.go (hunk 1 above)
Comment (mkrill) [reply framework/flamingo/sessions.go:60]: Hello Karsten, 
thx for your quick feedback. I updated ReadmeSession.md and added an entry in CueConfig(). Is there any reason, why the `?` for indicating optional parameters wasn't used so far?

Comment (mkrill) [reply framework/flamingo/sessions.go:60]: @karstenkoehler Regarding `CueConfig()` there is a second option: `keyPrefix: string | ""`. Please let me know, if you prefer the other one.

File: framework/flamingo/sessions.go [hunk 2]
```diff
@@ -76,6 +79,10 @@ func (m *SessionModule) Inject(config *struct {
 	m.redisClusterMode = config.RedisClusterMode
 	m.healthcheckSession = config.CheckSession
 
+	if strings.TrimSpace(config.RedisKeyPrefix) != "" {
+		m.redisKeyPrefix = &config.RedisKeyPrefix
```
Comment (karstenkoehler) [inline framework/flamingo/sessions.go:83]: @mkrill We should also trim the value here before assigning it.

Comment (mkrill) [reply framework/flamingo/sessions.go]: Done. Might also make sense for other parameters in the future, probably.

**Review by karstenkoehler — APPROVED** [review]

**Pull request merged**